framework/validators/ExistValidator.php

```diff
@@ -73,6 +73,11 @@ class ExistValidator extends Validator
      */
     public $targetAttributeJunction = 'and';
 
+    /**
+     * @var bool whether this validator can run on salve connections
+     */
+    public $canValidateOnSlaveDb =  true;
```

Comment:
@samdark the default value now is set to false, that mans validation runs on master by default